aws/data_source_aws_vpc_peering_connection_test.go

```diff
@@ -41,6 +41,25 @@ func TestAccDataSourceAwsVpcPeeringConnection_basic(t *testing.T) {
 	})
 }
 
+func TestAccDataSourceAwsVpcPeeringConnection_cidBlockSets(t *testing.T) {
```

Comment:
Nit: Rename to `TestAccDataSourceAwsVpcPeeringConnection_cidrBlockSets()`.

Reply:
Good catch!
Renamed